.github/workflows/ci.yml

```diff
@@ -79,6 +79,8 @@ jobs:
           noPrefix: true
           awsAccessKeyId: ${{ secrets.AWS_ACCESS_KEY_ID }}
           awsSecretAccessKey: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          githubToken: ${{ secrets.GITHUB_TOKEN }}
+          deployRunUrl: https://building-models-app.concord.org/__deployPath__/
```

Comment:
@dougmartin is this a good URL to test branches of sage at? 
Or would this be better:
`https://sagemodeler.concord.org/__deployPath__/app/`

Reply:
This is the correct one - sagemodeler.concord.org is a wrapper around this app running in CODAP.